app/assets/javascripts/templates/column.ejs

```diff
@@ -1,4 +1,4 @@
-<div class="column_header">
+<div class="column_header step-<%= id.replace("_", "-") %>">
```

Comment:
Why not use a data attribute `data-step="<%= id.replace("_", "-") %>"`?, it seems much cleaner, HTML classes should not be used to holding data, they're for styling 😄 

Also, it would be good to have a method on the column model to remove this `replace` from the template, something like `htmlFriendlyId()`